src/analyzerconsole.cpp

```diff
@@ -125,29 +125,29 @@ void CAnalyzerConsole::DrawErrorRateTrace()
     QPainter GraphPainter ( &GraphImage );
 
     // get the network buffer error rates to be displayed
-    CVector<double> vecButErrorRates;
-    double          dLimit;
-    double          dMaxUpLimit;
+    CVector<float>  vecButErrorRates;
```

Comment:
Why do you change everything from double to float? I think that is not necessary. Of course, the signal processing should be float, agreed, but please keep the changes to a minimum. So I do not see a need to change everything.

Reply:
It was just simpler when using "sed" to cover everything double, but I've reverted those chunks not related to audio now.